.github/report-flaky-tests/index.js

```diff
@@ -94,7 +94,7 @@ const metaData = {
 				)
 				.split(
 					new RegExp(
-						`(?<=${ TEST_RESULT.close })\n(?=${ TEST_RESULT.open })`
+						`(?<=${ TEST_RESULT.close })\n(?:...\n)?(?=${ TEST_RESULT.open })`
```

Comment:
It'd be great to have an example of the text this should split against.

BTW, I think the `.` will match any character, should it be `\.`?

Reply:
Oh right, I forgot about that!